src/main/webapp/app/shared/components/course-exam-archive-button/course-exam-archive-button.component.scss

```diff
@@ -0,0 +1,8 @@
+button[data-mode='Course'] {
```

Comment:
Can you give the button an id and use it here? I think it would be better to associate these styles only with this specific button instead of buttons in general.

Reply:
Done, thanks for the hint!